style.css

```diff
@@ -1,116 +1,133 @@
-.container {
+
+.navitems {
   display: flex;
-  flex-direction: row;
-  flex-wrap: nowrap;
-  justify-content: center;
+  list-style: none;
+}
 
+.rectangle1 {
   position: absolute;
-  width: 1440px;
-  height: 100px;
-  left: 0px;
-  top: 0px;
-  background: #FFFFFF;
-  box-shadow: 0px 0.5px 0px #DDDFE6;
-  }
+  width: 20px;
+  height: 2px;
+  right: 20px;
+  top: 19px;
+  background: #c4c4c4;
+}
 
-.nav-links{
-  display: flex;
-  list-style: none;  
+.rectangle2 {
+  position: absolute;
+  width: 20px;
+  height: 2px;
+  right: 20px;
+  top: 25px;
+  background: #c4c4c4;
 }
 
-.nav-item a {
-  display: flex;
-  padding: 10px 15px;
-  font-family: 'Poppins';
-  font-style: normal;
-  font-weight: 400;
-  font-size: 16px;
-  color: #1A2236;
-  text-decoration: none; 
-  text-align: right;
-  line-height: 24px;
-    
-  }
-  
-.nav-item:hover {
-  background-color: white;
-  }
-  
+.rectangle3 {
+  position: absolute;
+  width: 20px;
+  height: 2px;
+  right: 20px;
+  top: 31px;
+  background: #c4c4c4;
+}
 
+.rectangle1:hover {
+  background: white;
+}
+
+.rectangle2:hover {
+  background: white;
+}
+
+.rectangle3:hover {
+  background: white;
+}
 
 .container2 {
   position: absolute;
-  display:block;
-  height: 220px;
-  width: 484px;
-  left: 160px;
-  top: 292px;
-  border-radius: nullpx;
-  
-
-  font-family: 'Playfair Display';
+  display: flex;
+  height: 100px;
+  width: 220px;
+  left: 16px;
+  top: 70px;
+}
+
+.container2 h1 {
+  font-family: 'Playfair Display Bold', sans-serif;
   font-style: normal;
   font-weight: 400;
-  font-size: 110px;
+  font-size: 70px;
   line-height: 100%;
-/* or 110px */
-  color: #1A2236;
+
+  /* or 110px */
+  color: #1a2236;
 }
 
 .container3 {
-  display:flex;
+  display: flex;
   position: absolute;
-  width: 484px;
-  height: 84px;
-  left: 160px;
-  top: 760px;
+  width: 288px;
+  height: 126px;
+  left: 16px;
+  top: 250px;
+}
 
-  font-family: 'Poppins';
+.container3 p {
+  font-family: 'Poppins Thin', sans-serif;
   font-style: normal;
   font-weight: 400;
   font-size: 14px;
   line-height: 21px;
   letter-spacing: -0.01em;
-
-  color: #1A2236;
+  color: #1a2236;
 }
 
 .container4 {
   position: absolute;
-  width: 254px;
-  height: 70px;
-  left: 160px;
-  top: 900px;
-  
-    
-  background: #1A2236;
-
+  width: 288px;
+  height: 50px;
+  left: 16px;
+  top: 400px;
+  background: #1a2236;
 }
-.container4 p{
+
+.container4 p {
   position: absolute;
   width: 203.68px;
   height: 26.47px;
-  left: 30.16px;
-  top: 0px;
+  left: 50px;
+  top: 0;
+}
 
-  font-family: 'Poppins';
+.info p {
+  font-family: 'Poppins Thin', sans-serif;
```

Comment:
- [ ] Kindly fix the font family of your p tag according to the Figma template.

Reply:
when i use the font family as Poppins only i get an error in linters